components/bio-formats-plugins/src/loci/plugins/out/Exporter.java

```diff
@@ -299,14 +299,19 @@ public void run() {
             catch (DependencyException de) { }
             catch (ServiceException se) { }
 
+            OMEXMLMetadataRoot root = (OMEXMLMetadataRoot) store.getRoot();
+
             if (store == null) IJ.error("OME-XML Java library not found.");
+
+            if (root.sizeOfROIList()>0){
+                store.createRoot();
+            }
```

Comment:
This worries me a little, but I haven't yet tested thoroughly enough to know if it will cause a problem.